deploy/scripts/src/cri-kiwi/data/BAVdata.ts

```diff
@@ -0,0 +1,24 @@
+export const bankingPayload = JSON.stringify({
+  shared_claims: {
+    name: [
+      {
+        nameParts: [
+          {
+            value: 'Yasmine',
+            type: 'GivenName'
+          },
+          {
+            value: 'Young',
+            type: 'FamilyName'
+          }
+        ]
+      }
+    ],
+    birthDate: [
+      {
+        value: '1960-02-02'
+      }
+    ]
+  }
+}
+)
```

Comment:
Suggest adding a Type/Interface defintiion here for the object to ensure the payload is always of the correct format